src/components/action-buttons.component.tsx

```diff
@@ -23,15 +23,15 @@ import CloseDispenseForm from "../forms/close-dispense-form.component";
 interface ActionButtonsProps {
   medicationRequest: MedicationRequest;
   associatedMedicationDispenses: Array<MedicationDispense>;
-  mutate: Function;
```

Comment:
I refactored the mutate functionality to use a global mutate function instead of a bound mutate, so the mutate functionality has all been extracted out into utils.tsx, see my comments there.